app/models/job_invocation.rb

```diff
@@ -24,6 +24,13 @@ class JobInvocation < ActiveRecord::Base
 
   scoped_search :on => [:job_name], :complete_value => true
 
+  scoped_search :in => :last_task, :on => :started_at, :rename => 'last_task.started_at'
+  scoped_search :in => :last_task, :on => :ended_at, :rename => 'last_task.ended_at'
```

Comment:
Also add `:complete_value => true`, it adds some nice autocompletion for dates:

![complete](https://cloud.githubusercontent.com/assets/429763/11338097/134d0368-91c0-11e5-973a-1a63a6930f66.png)